_projects/use_coala.md

```diff
@@ -1,7 +1,8 @@
 ---
 collaborating_projects:
   - docker-coala-base
-desc: "Use coala CI on a popular GitHub project, of your choice, using a Docker image on Travis CI, and report the results to the coala developers."
+desc: "Use coala CI on a popular GitHub project, of your choice, using a Docker image on Travis CI, 
```

Comment:
break this line at 80cpl, and align indented text on next line under first letter of this line.